service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapperTest.java

```diff
@@ -193,6 +193,95 @@ public void When_MappingWithOccurrenceWithOnlyStart_Expect_TextDoesNotContainEar
         assertThat(actualXml).isEqualTo(expectedXml);
     }
 
+    @Test
+    public void When_MappingWithDeviceReferenceWhereDeviceHasManufacturer_Expect_TextContainsDeviceTypeTextAndManufacturer() {
+
+        var inputJson = """
+            {
+                "resourceType": "ProcedureRequest",
+                "id": "1B1C6F50-D8BE-4480-83D6-EF25AC5F1836",
+                "status": "active",
+                "intent": "plan",
+                "authoredOn": "2010-01-13T15:29:50.1+00:00",
+                "requester": {
+                    "agent": {
+                        "reference": "Device/6D340A1B-BC15-4D4E-93CF-BBCB5B74DF73"
+                    }
+                },
+                "code": {
+                    "text": "test"
+                }
+            }""";
+
+        final var expectedXml = """
+            <component typeCode="COMP" >
+                <PlanStatement classCode="OBS" moodCode="INT">
+                    <id root="394559384658936" />
+                    <code nullFlavor="UNK"><originalText>Mocked code</originalText></code>
+                    <text>
+                        Recall Device: Device type text Device name
```

Comment:
If I read the whole test, and nothing else, my assumption would be that the "Device type text" is "Device type text, and the "Device manufacturer" is "Device name". The first is a pretty easy assumption, but the second leaves me less confident.